### ISSUE TITLE
Loss and Accuracy dont change

### ISSUE THREAD

**Issue by OrkhanHI** [jiangtaoxie/fast-MPN-COV#15]
Hi, thank you for a great paper and code.

I have tried to run on Cars196 dataset the code where I used  *MPNCOV* and *mpnconvresnet50* in Jypyter Notebook, firsly I got error in #loss.backward()#. After adjusting the code with detach(), I could run the code. 

However, the loss and accuracy dont change from epoch to epoch, just oscillate around the same value. The best top 1 accuracy is 0.006 and the loss is just 5.28 for all epochs.

Could you help me to make the code run properly?

Thank you in advance.

**Comment by jiangtaoxie** [comment]
@OrkhanHI  Thanks, can you post the error details when you run the code with #loss.backward()# ?

**Comment by OrkhanHI** [comment]
This was the error I was getting, but I adjusted the code so model started to run, but acc and loss are the same.
![error](https://user-images.githubusercontent.com/46919226/58888203-a1344400-8719-11e9-81d6-f42fcc524674.PNG)
